src/pydvl/value/result.py

```diff
@@ -234,8 +235,12 @@ def __init__(
 
         self._algorithm = algorithm
         self._status = Status(status)  # Just in case we are given a string
-        self._values = values
-        self._variances = np.zeros_like(values) if variances is None else variances
+        self._values = arr_or_writeable_copy(values)
```

Comment:
Where have you encountered a problem with a non-writeable array? Also, the semantics now become a bit obscure. Copy or reuse? I could understand "always copy", or "always reuse", but mixing them can lead to errors...